js/NavMainComponent.js

```diff
@@ -74,8 +112,33 @@ NavMainComponent.prototype.switchSecondaryNav = function(target) {
     component.closeSubNavs();
 
     component.$html.find('.nav-list.is-active').removeClass('is-active');
-    component.$html.find('[data-nav="' + target + '"]')
-        .addClass('is-active');
+    component.$html.find('[data-nav="' + target + '"]').addClass('is-active');
+};
+
+NavMainComponent.prototype.switchTertiartyNav = function(target) {
+
+    var component = this;
+
+    component.$html.find('.navTertiary .nav-link').removeClass('is-active');
+
+    if (component.$html.find('[data-nav="' + target + '"]').length < 1) {
```

Comment:
We're making this query `component.$html.find('[data-nav="' + target + '"]')` twice in this function, once to get it's length and the second to add a class, could we store a reference to this query in a variable.